.travis.yml

```diff
@@ -11,6 +11,8 @@ rvm:
 script:
   - bundle exec rspec
   - bundle exec rubocop
+  - ./bin/daru-view -g
+  - ./bin/daru-view -H
```

Comment:
Can we have testcases like this : https://github.com/bundler/bundler/blob/master/spec/commands/info_spec.rb ?

Reply:
Yes, I have added tests in RSpec instead of Travis, please check.